.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,25 @@
-Please delete this line and the text below before submitting your contribution.
+# Pull Request checklist
 
----
+Thanks for contributing!
+Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.
+* * *
 
-Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.  
+## Tasks
 
-Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.  
+- [ ] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
+- [ ] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss what you are going to do.
+- [ ] Give this pull request a descriptive title.
+- [ ] If you are contributing to the lesson materials, make sure the content does not have spelling or grammar errors.
```

Comment:
Can we recommend a spellcheck app for markdown?

Reply:
Yeah that sounds like a good plan for tutorials in any case!

Reply:
@ledm Thanks. I could install Grammarly in VS Code. It is not very good as it should be. Shall suggest this?

Reply:
please see my [comment](https://github.com/ESMValGroup/tutorial/pull/19#issuecomment-601694273) .